### ISSUE TITLE
Help needed to install on Windows 7 32 bit.

### ISSUE THREAD

**Issue by cydside** [AllenDang/giu#147]
Hi to all,
sorry to ask that help but I'd like to use this amazing package on Win 7 32bit. So I installed mingw for gcc and g++ but still having issues running `go build widgets.go`.
What am I missing?
Thanks in advance.

**Comment by AllenDang** [comment]
@cydside What's the error message? You should run `go build .` under the root folder of giu.

**Comment by cydside** [comment]
I just copied the content of widgets.go in examples folder in imgui01.go file and here is the error output.

[error.txt](https://github.com/AllenDang/giu/files/6066106/error.txt)

**Comment by AllenDang** [comment]
@cydside What happens if you run `go get github.com/AllenDang/giu`?

**Comment by cydside** [comment]
Sorry for the late answer, I had to create a VM Win7 x86 and re-install all the software.
Running  `go get github.com/AllenDang/giu` i have:


C:\Dev\imgui01>go get -u github.com/AllenDang/giu
go: downloading github.com/go-gl/glfw v0.0.0-20201108214237-06ea97f0c265
go: downloading github.com/go-resty/resty/v2 v2.5.0
go: downloading github.com/go-resty/resty v1.12.0
go: downloading golang.org/x/net v0.0.0-20210226172049-e18ecbb05110
go build github.com/go-gl/gl/v3.2-core/gl: c:\go\pkg\tool\windows_386\cgo.exe: exit status 2
go build github.com/go-gl/glfw/v3.3/glfw: c:\go\pkg\tool\windows_386\cgo.exe: exit status 2

**Comment by cydside** [comment]
Doesn't work on VM due to OpenGL support that is only for version 2.x on Virtulbox using VboxSVGA. I ended to install a new real machine with Win 7 x86, Go 1.16 and Mingw 8.1.0. After that I tried to not install the Video drivers leaving the standard VGA and still had the panic error above mentioned. Once installed the drivers it worked. Sorry to keep you on this issue, hope only to be useful for others. Thanks
